vioscsi/vioscsi.c

```diff
@@ -358,10 +360,20 @@ DriverEntry(
     InitializeDebugPrints((PDRIVER_OBJECT)DriverObject, (PUNICODE_STRING)RegistryPath);
 
     IsCrashDumpMode = FALSE;
-    RhelDbgPrint(TRACE_LEVEL_FATAL, " Vioscsi driver started...built on %s %s\n", __DATE__, __TIME__);
+    #if !defined(RUN_UNCHECKED) || defined(RUN_MIN_CHECKED)
```

Comment:
Why are the printouts conditional? This is not hot path, print always.

Reply:
@YanVugenfirer 
I have started a discussion related to this over [here](https://github.com/virtio-win/kvm-guest-drivers-windows/pull/1214#issuecomment-2543969865).

Reply:
I'll be updating this per PR #1228, but note this particular event is never logged with ETW as it hasn't loaded yet, i.e. this is for DBG-enabled builds only.